### PR TITLE
Feat/covid form

### PR DESCRIPTION
**Pull request by hihilary**
1. add CovidDeclaration in FormRPartB
2. add FormSwitch and api to return to frontend if the specific functionality of some form is enabled.

### CODE REVIEW

File: src/main/java/uk/nhs/hee/tis/trainee/forms/dto/FormSwitchDto.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+package uk.nhs.hee.tis.trainee.forms.dto;
+
+import lombok.Data;
+
+@Data
+public class FormSwitchDto {
+
+  private String id;
+  String name;
+  boolean enabled;
```
Comment (Judge40) [inline src/main/java/uk/nhs/hee/tis/trainee/forms/dto/FormSwitchDto.java:31]: These should be private too.

File: src/main/java/uk/nhs/hee/tis/trainee/forms/model/FormSwitch.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+@Document(collection = "FormSwitch")
+@Data
+public class FormSwitch {
+  @Id
+  private String id;
+
+  String name;
+
+  boolean enabled;
```
Comment (Judge40) [inline src/main/java/uk/nhs/hee/tis/trainee/forms/model/FormSwitch.java:36]: Should be private.

File: src/main/java/uk/nhs/hee/tis/trainee/forms/dto/FormSwitchDto.java (hunk 1 above)
Comment (hihilary) [reply src/main/java/uk/nhs/hee/tis/trainee/forms/dto/FormSwitchDto.java]: Fixed. Cheers.

File: src/main/java/uk/nhs/hee/tis/trainee/forms/model/FormSwitch.java (hunk 2 above)
Comment (hihilary) [reply src/main/java/uk/nhs/hee/tis/trainee/forms/model/FormSwitch.java]: Fixed.

**Pull request merged**